README.md

```diff
@@ -1374,13 +1374,14 @@ _You can enable the following settings in Xcode by running [this script](resourc
   * `// MARK: Fileprivate` for `fileprivate` properties and methods.
   * `// MARK: Private` for `private` properties and methods.
   * If the type in question is an enum, its cases should go above the first `// MARK:`.
-  * If there are typealiases, they should go above the first `// MARK:`.
+  * If there are typealiases or nested types, they should go above the first `// MARK:`.
```

Comment:
This rule previously did not specify where we should place nested types.

I think they we should place them at the top, above the first `// MARK:`. This matches the existing policy for `typealias` declarations.

Reply:
This makes sense to me.

Reply:
How does this handle types that have a specific access control level? I'd be concerned if nested private types were grouped with public types

Reply:
This places all nested types and typealiases (regardless of access control) above the first `// MARK:`. 

I personally think this is reasonable, but I can see the argument against putting `private` declarations at the top of a type.

Reply:
[`d0d660e`](https://github.com/airbnb/swift/pull/102/commits/d0d660e644569894d67d5787d414f7c18f074b38) removes this exception in favor of grouping nested types at the top of their respective access-control category.

Reply:
I think that's a good change.